db/migrate/20221122202416_create_users.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Database migration implementation
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.string :posts_counter
```

Comment:
- [ ] `posts_counter` should be of the data type integer, kindly fix this

Reply:
Noted and resolved